cmd/tableprinters/image.go

```diff
@@ -15,10 +15,7 @@ func (t *TablePrinter) ImageTable(data []*models.V1ImageResponse, wide bool) ([]
 		rows [][]string
 	)
 
-	header := []string{"ID", "Name", "Description", "Features", "Expiration", "Status"}
-	if wide {
-		header = []string{"ID", "Name", "Description", "Features", "Expiration", "Status", "UsedBy"}
-	}
+	header := []string{"ID", "Name", "Description", "Features", "Expiration", "Status", "UsedBy"}
 
 	sort.SliceStable(data, func(i, j int) bool { return *data[i].ID < *data[j].ID })
```

Comment:
I think this line could even be removed, too.

Reply:
done